docs/releases/2.0.txt

```diff
@@ -54,6 +54,8 @@ Minor features
 * The default iteration count for the PBKDF2 password hasher is increased from
   36,000 to 100,000.
 
+* The maximum length of `User.last_name` is increased to 100 characters.
```

Comment:
This wasn't considered a "feature" in 15ef1dd478c5b6f005e5f782b7619f718ed55e84 so I think we can also omit it here.